jupyterhub/notebook-images/overlays/additional/minimal-notebook-imagestream.yaml

```
@@ -22,7 +22,7 @@ spec:
     from:
       kind: DockerImage
       # in a downstream build "latest" will be replaced
-      name: quay.io/modh/odh-minimal-notebook-container:latest
+      name: quay.io/modh/odh-minimal-notebook-container@sha256:34d396306badc7cb96af2cac1181f840b73f5bfbd87a053735782faa2f67a3df
```

https://github.com/red-hat-data-services/odh-manifests/pull/248 has been merged. Can you update this image and the sha256 in the live builder PR?

```suggestion
      name: quay.io/modh/odh-minimal-notebook-container@sha256:a5a7738b09a204804e084a45f96360b568b0b9d85709c0ce6742d440ff917183
```

---

Updated live builder too.